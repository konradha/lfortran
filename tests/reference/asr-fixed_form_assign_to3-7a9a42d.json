{
    "basename": "asr-fixed_form_assign_to3-7a9a42d",
    "cmd": "lfortran --indent --fixed-form --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form_assign_to3.f",
    "infile_hash": "cef165b367ac19491dbe0628a23d1843039e77602c05b5554ae12997",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-fixed_form_assign_to3-7a9a42d.stdout",
    "stdout_hash": "abf79208b1cdc534419c49a65e135502cd0f815883f19fdbd45fef4c",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}