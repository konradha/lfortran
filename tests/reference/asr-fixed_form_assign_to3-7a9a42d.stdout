(TranslationUnit 
    (SymbolTable
        1
        {
            assign3:
                (Program 
                    (SymbolTable
                        2
                        {
                            next:
                                (Variable 
                                    2 
                                    next 
                                    Local 
                                    () 
                                    () 
                                    Default 
                                    (Integer 8 []) 
                                    Source 
                                    Public 
                                    Optional 
                                    .false.
                                )
                            
                        }) 
                    assign3 
                    [] 
                    [(GoToTarget 
                        10
                    )
                    (= 
                        (Var 2 next) 
                        (IntegerConstant 30 (Integer 8 [])) 
                        ()
                    )
                    (GoToTarget 
                        20
                    )
                    (Select 
                        (IntegerConstant 0 (Integer 8 [])) 
                        [(CaseStmt 
                            [(IntegerConstant 30 (Integer 4 []))] 
                            [(GoTo 
                                30
                            )]
                        )
                        (CaseStmt 
                            [(IntegerConstant 50 (Integer 4 []))] 
                            [(GoTo 
                                50
                            )]
                        )
                        (CaseStmt 
                            [(IntegerConstant 70 (Integer 4 []))] 
                            [(GoTo 
                                70
                            )]
                        )
                        (CaseStmt 
                            [(IntegerConstant 110 (Integer 4 []))] 
                            [(GoTo 
                                110
                            )]
                        )] 
                        []
                    )
                    (= 
                        (Var 2 next) 
                        (IntegerConstant 50 (Integer 8 [])) 
                        ()
                    )
                    (= 
                        (Var 2 next) 
                        (IntegerConstant 70 (Integer 8 [])) 
                        ()
                    )
                    (= 
                        (Var 2 next) 
                        (IntegerConstant 110 (Integer 8 [])) 
                        ()
                    )
                    (GoToTarget 
                        30
                    )
                    (Print 
                        () 
                        [(IntegerConstant 30 (Integer 4 []))] 
                        () 
                        ()
                    )
                    (GoToTarget 
                        50
                    )
                    (Print 
                        () 
                        [(IntegerConstant 50 (Integer 4 []))] 
                        () 
                        ()
                    )
                    (GoToTarget 
                        70
                    )
                    (Print 
                        () 
                        [(IntegerConstant 70 (Integer 4 []))] 
                        () 
                        ()
                    )
                    (GoToTarget 
                        110
                    )
                    (Print 
                        () 
                        [(IntegerConstant 110 (Integer 4 []))] 
                        () 
                        ()
                    )]
                )
            
        }) 
    []
)
